jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/TargetImageParameters.java

```diff
@@ -34,42 +36,51 @@
 
   private final AuthParameters auth;
 
-  @Nullable private String image;
-  private Set<String> tags = Collections.emptySet();
+  private Property<String> image;
+  private SetProperty<String> tags;
   @Nullable private String credHelper;
 
   @Inject
   public TargetImageParameters(ObjectFactory objectFactory) {
     auth = objectFactory.newInstance(AuthParameters.class, "to.auth");
+    image = objectFactory.property(String.class);
+    tags = objectFactory.setProperty(String.class);
```

Comment:
Maybe it's safer to initialize it to an empty set like we do it here:

https://github.com/GoogleContainerTools/jib/blob/94f8bdafe0f816c548f96392b34660d5574cc5c4/jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/JibExtension.java#L116